gtda/base.py

```diff
@@ -139,7 +139,7 @@ def transform_plot(self, X, sample=0, **plot_params):
             Transformed one-sample slice from the input.
 
         """
-        Xt = self.transform(X[[sample]])
+        Xt = self.transform(X[sample:sample+1])
```

Comment:
I am very surprised :D

Reply:
It was required here because the existing approach cannot cover lists. It could become the accepted way "just in case", but for now we use arrays everywhere other than in `homology/simplicial.py` so changing things is not urgent in my opinion.